### DISCUSSION TITLE
why V3.5.7 Script Validation Error？

### DISCUSSION THREAD

**Discussion by cwk0099** [naver/ngrinder#942] [Q&A]
os: amazon linux
How to run ngrinder：docker
Steps to reproduce the bug：always when click validate button

response in /script/api/validate
{
"message" : "Handler dispatch failed; nested exception is java.lang.NoClassDefFoundError: Could not initialize class org.ngrinder.common.util.SystemInfoUtils",
"success" : false,
"stackTrace" : "org.springframework.web.util.NestedServletException: Handler dispatch failed; nested exception is java.lang.NoClassDefFoundError: Could not initialize class org.ngrinder.common.util.SystemInfoUtils\n\tat org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)\n\tat org.ngrinder.security.PluggablePreAuthFilter.doFilter(PluggablePreAuthFilter.java:52)\n\tat org.ngrinder.infra.servlet.PluggableServletFilter.doFilter(PluggableServletFilter.java:61)\n\tat org.ngrinder.security.PluggablePreAuthFilter.doFilter(PluggablePreAuthFilter.java:52)\n\tat org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)\n\tat org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)\n\tat org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)\nCaused by: java.lang.NoClassDefFoundError: Could not initialize class org.ngrinder.common.util.SystemInfoUtils\n\tat net.grinder.engine.agent.PropertyBuilder.buildJVMArgumentWithoutMemory(PropertyBuilder.java:208)\n\tat net.grinder.engine.agent.LocalScriptTestDriveService.doValidate(LocalScriptTestDriveService.java:131)\n\tat org.ngrinder.script.service.ScriptValidationService.validate(ScriptValidationService.java:117)\n\tat org.ngrinder.script.controller.FileEntryApiController.validate(FileEntryApiController.java:392)\n\t... 7 more\n"
}

**Comment by junoyoon** [comment]
@donggyu04 please check it.
